achievements/contribubit.achievement.js

```diff
@@ -0,0 +1,24 @@
+var _ = require('lodash');
+var console = process.console;
+
+var TheGodfatherConsigliere = {
+	name: 'The Godfather Consigliere',
+	check: function(pullRequest, shall) {
+		if (_.result(pullRequest, 'organization.username') === 'Kibibit') {
+			var achievement = {
+				avatar : 'images/achievements/thegodfather.achievement.jpg',
+				name: 'The Godfather Consigliere',
+				short: 'Great men are not born great, they contribute to Kibibit . . . ',
+				description: 'You have contributed to a Kibibit repository. Accept this achievement as gift on my daughter\'s wedding day',
```

Comment:
maybe we want to sound more excited in the first sentence. I would also change `contributed to a Kibibit repository` to just say `contributed to Kibibit`

maybe something about the fact that we appreciate your contribution?

I think:
```
'You have contributed to Kibibit! Thanks a lot!<br>Accept this achievement as gift on my daughter\'s wedding day'
```
or
```
'You have contributed to Kibibit! We really appreciate it!<br>Accept this achievement as gift on my daughter\'s wedding day'
```